cpp/dolfinx/geometry/BoundingBoxTree.cpp

```diff
@@ -39,23 +40,27 @@ compute_bbox_of_entity(const mesh::Mesh& mesh, int dim, std::int32_t index)
 {
   // Get the geometrical indices for the mesh entity
   const int tdim = mesh.topology().dim();
-  const Eigen::Array<double, Eigen::Dynamic, 3, Eigen::RowMajor>& geom_dofs
-      = mesh.geometry().x();
+  // Use eigen map for now.
```

Comment:
Are these meant to be more like `FIXME` notes?

Reply:
Yes. I'll add `FIXME` flag.